pkgs/by-name/po/poutine/package.nix

```diff
@@ -15,13 +17,22 @@ buildGoModule rec {
     hash = "sha256-T81Qi79VaZzfKL4niTZQW+gwwiBcyInALrvyUg1V4Ck=";
   };
 
+  nativeBuildInputs = [ installShellFiles ];
```

Comment:
`nativeBuildInputs` is out of phases order.

https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases

`vendorHash` is unpackPhase (it's a bundle of deps, agree?)
`ldflags` is configurePhase
`nativeBuildInputs` is buildPhase